src/sonic-yang-models/yang-models/sonic-bgp-neighbor.yang

```diff
@@ -97,6 +97,11 @@ module sonic-bgp-neighbor {
                     }
                     description "Route reflector client";
                 }
+
+                leaf admin_status {
+                    type boolean;
+                    description "To enable BGP peer";
```

Comment:
>To enable BGP peer [](http://example.com/codeflow?start=33&length=18)

We can "enable" a BGP peer but keep it as admin down. How about "Admin status of BGP peer"?